player_openal.go

```diff
@@ -45,12 +45,13 @@ type player struct {
 	// alContext represents a pointer to ALCcontext. The type is uintptr since the value
 	// can be 0x18 on macOS, which is invalid as a pointer value, and this might cause
 	// GC errors.
-	alContext  uintptr
-	alDevice   uintptr
-	alSource   C.ALuint
-	sampleRate int
-	isClosed   bool
-	alFormat   C.ALenum
+	alContext    uintptr
+	alDevice     *C.ALCdevice
```

Comment:
alDevice can be also invalid value as pointer on macOS. This causes flaky crash.